docroot/themes/humsci/humsci_basic/src/scss/elements/_tables.scss

```diff
@@ -0,0 +1,52 @@
+.hb-table-wrap {
+  margin: auto;
```

Comment:
I have a question about the mobile styles for this... readme me when we pair! I wondered if you used kasey's codepen